Kitodo/src/main/java/org/kitodo/export/ExportDms.java

```diff
@@ -90,16 +96,21 @@ public boolean startExport(Process process) throws DataException {
         boolean wasNotAlreadyExported = !process.isExported();
         if (wasNotAlreadyExported) {
             process.setExported(true);
-            ServiceManager.getProcessService().save(process);
+            processService.save(process);
+        }
+
+        if (!exportCompletedChildren(process.getChildren())) {
```

Comment:
If an error occures on child export, the process is still saved as "exported"

Reply:
Good catch. Needs fixing